src/tests/modules/ordered.rs

```diff
@@ -48,17 +51,16 @@ fn ordered_insert_shorthand() {
 
 #[cfg(feature = "decimal128")]
 fn test_decimal128(doc: &mut Document) {
```

Comment:
I think `#[test]` needs to be added above this test -- clippy is complaining that it doesn't get used.

Reply:
I think clippy was complaining since I removed the call to this test. Added it back with the decimal128 feature enabled. So it shouldn't complain now